modules/ballerina-http/src/main/ballerina/ballerina/net/http/annotation.bal

```diff
@@ -53,12 +53,12 @@ public annotation webSocket attach service<> {
 }
 
 @Description { value:"Configuration for HTTP resource"}
-@Field {value:"methods: The array of HTTP methods"}
+@Field {value:"methods: The array of allowed HTTP methods"}
 @Field {value:"path: The path of resource"}
 @Field {value:"consumes: The media types which are accepted by resource"}
 @Field {value:"produces: The media types which are produced by resource"}
-@Field {value:"allowOrigins: The array of origins which the response is shared by the resource"}
-@Field {value:"allowCredentials: Enables the need of credentials to the resource"}
+@Field {value:"allowOrigins: The array of origins with which the response is shared by the resource"}
+@Field {value:"allowCredentials: Specifies whether credentials are required to access the service"}
```

Comment:
Shall we change this to "access the resource" instead of "access the service". Because resource level CORS configurations only define the resource attributes

Reply:
Ack.